code/game/objects/items/devices/busterarm/_buster.dm

```diff
@@ -47,3 +47,13 @@
 		var/obj/item/bodypart/limb_to_hit = target.get_bodypart(user.zone_selected)
 		var/armor = target.run_armor_check(limb_to_hit, MELEE, armour_penetration = 35)
 		target.apply_damage(damage, BRUTE, limb_to_hit, armor, wound_bonus=CANT_WOUND)
+
+//knocking them down
+/datum/action/cooldown/buster/proc/footsies(mob/living/target)
+	if(target.stat == CONSCIOUS && target.resting == FALSE && !target.has_status_effect(STATUS_EFFECT_PARALYZED) && !target.has_status_effect(STATUS_EFFECT_KNOCKDOWN))
+		animate(target, transform = matrix(90, MATRIX_ROTATE), time = 0.1 SECONDS, loop = 0)
+
+//Check for if someone is allowed to be stood back up
+/datum/action/cooldown/buster/proc/wakeup(mob/living/target)
+	if(target.stat == CONSCIOUS && target.resting == FALSE && !target.has_status_effect(STATUS_EFFECT_PARALYZED) && !target.has_status_effect(STATUS_EFFECT_KNOCKDOWN))
```

Comment:
Is there a reason you don't just use `MOBILITY_STAND` to knock people over and detect if they're on the floor?

Reply:
hadnt heard of it before today

Reply:
need to hold off on the pr because this is acting up a bit